scripts/monitoring/auto_scale_workers.py

```diff
@@ -0,0 +1,136 @@
+import os
+import time
+import pytz
+import requests
+
+import boto3
+
+import botocore
+
+from auto_stop_workers import start_worker, stop_worker
+
+utc = pytz.UTC
+
+
+# Eval AI related credentials
+evalai_endpoint = os.environ.get("API_HOST_URL")
+authorization_header = {
+    "Authorization": "Bearer {}".format(os.environ.get("AUTH_TOKEN"))
+}
+
+
+def execute_get_request(url):
+    response = requests.get(url, headers=authorization_header)
+    return response.json()
+
+
+def get_challenges():
+    all_challenge_endpoint = "{}/api/challenges/challenge/all/all/all".format(
+        evalai_endpoint  # Gets all challenges
+    )
+    response = execute_get_request(all_challenge_endpoint)
+
+    return response.json()
+
+
+def get_sqs_queue_by_name(queue_name):
+    """
+    Returns:
+        Returns the SQS Queue object
+    """
+
+    sqs = boto3.resource(
+        "sqs",
+        region_name=os.environ.get("AWS_DEFAULT_REGION", "us-east-1"),
+        aws_secret_access_key=os.environ.get("AWS_SECRET_ACCESS_KEY"),
+        aws_access_key_id=os.environ.get("AWS_ACCESS_KEY_ID"),
+    )
+    if queue_name == "":
+        raise ValueError("Queue Name cannot be empty")
+
+    try:
+        queue = sqs.get_queue_by_name(QueueName=queue_name)
+    except botocore.exceptions.ClientError as ex:
+        if (
+            ex.response["Error"]["Code"]
+            != "AWS.SimpleQueueService.NonExistentQueue"
+        ):
+            raise Exception("Cannot get queue: {}".format(queue_name))
+    return queue
+
+
+def get_queue_length(queue):
+    # TODO: Check if we should use approximate number of messages
+    # https://github.com/boto/boto3/issues/599
+    # Ref: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/sqs.html#using-an-existing-queue
+    # Ref: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Queue.attributes
+    return (
+        queue.attributes["ApproximateNumberOfMessages"]
+        + queue.attributes["ApproximateNumberOfMessages"]
```

Comment:
you are adding exact same values here? This is a typo I guess. Can you please fix this?